include/mbedtls/ssl_internal.h

```diff
@@ -507,20 +591,21 @@ int mbedtls_ssl_fetch_input( mbedtls_ssl_context *ssl, size_t nb_want );
 int mbedtls_ssl_write_record( mbedtls_ssl_context *ssl );
 int mbedtls_ssl_flush_output( mbedtls_ssl_context *ssl );
 
-int mbedtls_ssl_parse_certificate( mbedtls_ssl_context *ssl );
-int mbedtls_ssl_write_certificate( mbedtls_ssl_context *ssl );
+int mbedtls_ssl_process_read_certificate( mbedtls_ssl_context *ssl );
+int mbedtls_ssl_process_write_certificate( mbedtls_ssl_context *ssl );
 
-int mbedtls_ssl_parse_change_cipher_spec( mbedtls_ssl_context *ssl );
-int mbedtls_ssl_write_change_cipher_spec( mbedtls_ssl_context *ssl );
+int mbedtls_ssl_process_out_ccs( mbedtls_ssl_context *ssl );
```

Comment:
It would be good to know what "ccs" stands for.

Reply:
Done.